docker-compose.yml

```diff
@@ -1,33 +1,45 @@
-version: '2'
+version: '3'
 
 services:
   mdb:
     image: mongo:3.4.3
     ports:
       - "27017"
     command: mongod
-
+  bdb-base:
+    image: bigchaindb
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        backend: localmongodb
+    volumes:
+      - ./bigchaindb:/usr/src/app/bigchaindb
+      - ./tests:/usr/src/app/tests
   bdb:
+    depends_on:
+      - mdb
+      - tendermint
     build:
       context: .
       dockerfile: Dockerfile-dev
-      args:
-        backend: mongodb
     volumes:
       - ./bigchaindb:/usr/src/app/bigchaindb
       - ./tests:/usr/src/app/tests
-      - ./docs:/usr/src/app/docs
-      - ./k8s:/usr/src/app/k8s
-      - ./setup.py:/usr/src/app/setup.py
-      - ./setup.cfg:/usr/src/app/setup.cfg
-      - ./pytest.ini:/usr/src/app/pytest.ini
-      - ./tox.ini:/usr/src/app/tox.ini
     environment:
-      BIGCHAINDB_DATABASE_BACKEND: localmongodb
```

Comment:
Why is this removed?

Reply:
we are already passing that as an arg no need for env variable too